script.js

```diff
@@ -0,0 +1,177 @@
+        const cTempRound = Math.round(currentWeather.temp_c)
+        const fTempRound = Math.round(currentWeather.temp_f)
+        weatherDisplay.className = 'weatherDisplay'
+        weatherDisplay.innerHTML = `
+            <h5>${locationDetails?.name || ''},  ${locationDetails?.country || ''}</h5>
+            <h3 class= 'current-temp-c'>${cTempRound}°c</h3>
+            <h3 class= 'current-temp-f'>${fTempRound}°f</h3>
+            <img src='${currentWeather.condition.icon}'>
+            <h4>${currentWeather.condition.text}</h4>
+            <h4>${locationDetails?.localtime || ''}</h4>`
+        weatherResults.appendChild(weatherDisplay)
+    }  
+    if (weatherInfo.forecast) {
+        forecastdays = weatherInfo.forecast.forecastday
+        locationDetails = weatherInfo?.location;
+                forecastdays.forEach((dayReport) => {
+            dayTemp = dayReport.day
+            const weatherDisplay = document.createElement('div')
+            weatherDisplay.className = 'weatherDisplay'
+            const cTempHRound = Math.round(dayTemp.maxtemp_c)
+            const cTempLRound = Math.round(dayTemp.mintemp_c)
+            const fTempHRound = Math.round(dayTemp.maxtemp_f)
+            const fTempLRound = Math.round(dayTemp.mintemp_f)
+            weatherDisplay.innerHTML = `
+                <h5>${locationDetails?.name || ''},  ${locationDetails?.country || ''}</h5>
+                <img src='${dayReport.day.condition.icon}'>
+                <h4>${dayReport.day.condition.text}</h4>
+                <h4>${dayReport.date}</h4>
+                <div class='c-temp'>
+                    <h3 class='temp-c'>H: ${cTempHRound}°c</h3>
+                    <h3 class='temp-c'>L: ${cTempLRound}°c</h3>
+                    </div>
+                    <div class='f-temp'>
+                    <h3 class='temp-f'>H: ${fTempHRound}°f</h3>
+                    <h3 class='temp-f'>L: ${fTempLRound}°f</h3>
+                </div>`
+            weatherResults.appendChild(weatherDisplay)
+        })     
+    }
+    toggleTemp()
+}
+
+function onError(error){
+    const container = document.querySelector('.container')
+    const errorDisplay = document.createElement('h3')
+    errorDisplay.className = 'error'
+    errorDisplay.innerText = error
+    errorDisplay.style.color = 'red'
+    container.appendChild(errorDisplay)
+}
+
+function errorDisplay(){
+    const errorDisplay = document.querySelector('.error')
+    if (errorDisplay){
+        errorDisplay.remove()
+    }
+}
+
+function clearSearch(){
+    form = document.querySelector('form')
+    const days = document.querySelector('.num-days')
+    form.reset()
+    days.style.display = 'none'
+}
+
+function resetDisplay(){
+    const weatherDisplayDiv = document.querySelector('.weatherDisplay')
+    if(weatherDisplayDiv){
+        weatherDisplayDiv.remove()
+    }
+}
+
+function toggleTemp(){
+    const cTemp = document.querySelectorAll('.c-temp')
```

Comment:
This function can be split into 4 different functions for handling the click events

Reply:
I’ll affect these corrections